packages/dapp/src/components/Dashboard/ManageNewsroom/WithNewsroomChannelAdminList.tsx

```diff
@@ -33,6 +33,16 @@ export interface WithNewsroomChannelAdminListProps {
   children(props: { newsroomAddresses: Set<EthAddress> }): any;
 }
 
+export function newsroomChannelsFromQueryData(data?: any): EthAddress[] {
+  if (data && data.currentUser && data.currentUser.channels && data.currentUser.channels.filter) {
+    return data.currentUser.channels
+      .filter((channelMember: any) => channelMember.channel.channelType === "newsroom")
```

Comment:
don't we also need to filter on whether or not the channelMember is an admin?

Reply:
Good question! I thought that the `currentUser { channels { ... } }` query only returned channels that the current user is an admin of, but if not then yes we should filter by `currentUserIsAdmin`. Can you or @dankins confirm the behavior?

Reply:
Just checked the query and I don't see anything that only returns channelMember results of type admin. But since your query already gets the `role` you can just check if that is `admin`, I believe